recipes/seaborn/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.7.0" %}
+{% set name= "seaborn" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 764c030a9db60ec804b9a506d0a38cd2
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - matplotlib
+    - setuptools
+  run:
+    - python
+    - numpy
```

Comment:
Only if they are linking to it, which I don't think is the case with `seaborn` as it is pure Python.